File: problem_2/solve_2.js
function main() {
    i = 0
    prev = 0
    next = 1
    tmp = 0

    while (prev + next < 4000000) {
        tmp = next + prev
        if (tmp % 2 == 0)
            i += tmp
        prev = next
        next = tmp
    }

    console.log(i)
}

main()

File: problem_2/solve_2.c
#include <stdio.h>

int main() {
    int i = 0;
    int prev = 0;
    int next = 1;
    int tmp = 0;

    while (prev + next < 4000000) {
        tmp = prev + next;
        if (tmp % 2 == 0)
            i += tmp;
        prev = next;
        next = tmp;
    }

    printf("%d", i);
}